stellarphot/settings/tests/test_models.py

```diff
@@ -29,3 +33,43 @@ def test_create_invalid_values(bad_one):
     bad_settings[bad_one] = -1
     with pytest.raises(ValidationError, match=bad_one):
         ApertureSettings(**bad_settings)
+
+
+DEFAULT_EXOPLANET_SETTINGS = dict(
+    epoch=Time(0, format="jd"),
+    period=0 * u.min,
+    identifier="",
+    coordinate=SkyCoord(
+        ra="00:00:00.00", dec="+00:00:00.0", frame="icrs",
+        unit=("hour", "degree")
+    ),
+    depth=0,
+    duration=0 * u.min,
+)
+
+
+def test_create_exoplanet_correctly():
+    planet = Exoplanet(**DEFAULT_EXOPLANET_SETTINGS)
+    assert planet.epoch == DEFAULT_EXOPLANET_SETTINGS["epoch"]
+    assert u.get_physical_type(planet.period) == "time"
+    assert planet.identifier == DEFAULT_EXOPLANET_SETTINGS["identifier"]
+    assert planet.coordinate == DEFAULT_EXOPLANET_SETTINGS["coordinate"]
+    assert planet.depth == DEFAULT_EXOPLANET_SETTINGS["depth"]
+    assert u.get_physical_type(planet.duration) == "time"
+
+
+@pytest.mark.parametrize("bad_one", [
+    {
+        "epoch": 0,
+        "period": 120,
+        "identifier": "bad-planet",
+        "coordinate": SkyCoord(ra="00:00:00.00", dec="+00:00:00.0",
+                               frame="icrs", unit=("hour", "degree")),
+        "depth": 0,
+        "duration": 42 * u.C,
+    }
+])
+def test_create_invalid_exoplanet(bad_one):
+    # Check that individual values that are bad raise an error
+    with pytest.raises(ValidationError, match=".*"):
```

Comment:
Typically if `match` is included you should try to match some meaningful part of the error message. This particular value `".*"` means it will always match, which is equivalent to not having `match` there at all.